voice_insights_summary.go

```diff
@@ -0,0 +1,154 @@
+package twilio
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+)
+
+const SummaryPathPart = "Summary"
+
+type CallSummaryService struct {
+	callSid string
+	client  *Client
+}
+
+type CallSummary struct {
+	AccountSid      string            `json:"account_sid"`
+	Attributes      Attributes        `json:"attributes"`
+	CallSid         string            `json:"call_sid"`
+	CallState       Status            `json:"call_state"`
+	CallType        string            `json:"call_type"`
+	CarrierEdge     *Edge             `json:"carrier_edge,omitempty"`
+	ClientEdge      *Edge             `json:"client_edge,omitempty"`
+	ConnectDuration int               `json:"connect_duration"`
+	Duration        int               `json:"duration"`
+	EndTime         TwilioTime        `json:"end_time"`
+	From            ToFrom            `json:"from"`
+	ProcessingState string            `json:"processing_state"`
+	Properties      CallProperties    `json:"properties"`
+	SdkEdge         *SdkEdge          `json:"sdk_edge,omitempty"`
+	SipEdge         *Edge             `json:"sip_edge,omitempty"`
+	StartTime       TwilioTime        `json:"start_time"`
+	Tags            map[string]string `json:"tags"`
+	To              ToFrom            `json:"to"`
+	URL             string            `json:"url"`
+}
+
+type Attributes struct {
```

Comment:
Not thrilled about having `Attributes` as a top level global that refers specifically to the VoiceInsights... how about `CallSummaryAttributes`?